### DISCUSSION TITLE
Does longhorn support manage puredisk without mount to a system path?

### DISCUSSION THREAD

**Discussion by leadersnowy** [longhorn/longhorn#3467] [Q&A]
Hello,

I need some help.

I‘d like to know if Longhorn supports direct management of pure block storage devices like ceph, instead of mount --bind the devices to a system path.

Thank you

**Comment by derekbit** [comment]
@leadersnowy,
Longhorn needs to rely on the underlying filesystem and its features to manage and operate the replicas.
It does not support the direct management of pure block storage devices, so you have to format and mount your block device first.

**Reply by leadersnowy** [reply]
OK, thanks!
